src/composables/useTokenLists.ts

```diff
@@ -75,13 +75,16 @@ export default function useTokenLists(request?: TokenListRequest) {
   const queryKey = QUERY_KEYS.TokenLists;
   const queryFn = loadAllTokenLists;
 
-  const { data: lists, isLoading } = useQuery<TokenList[]>(queryKey, queryFn);
+  const { data: lists, isLoading } = useQuery<TokenList[]>(queryKey, queryFn, {
+    refetchOnMount: false,
+    refetchOnWindowFocus: false
```

Comment:
To further optimize loading, I was thinking its best to avoid refetching when a component mounts / window refocus for this specific query.

Token lists don't change that often and I think we could think of a better mechanism to update them throughout the user session.